### DISCUSSION TITLE
Helm production parameters Quorum size, why are theses values set to 1

### DISCUSSION THREAD

**Discussion by gmatej** [apache/pulsar#23528] [Q&A]
Hi.

We are looking at the helm production charts for Pulsar and also Bookkeeper. Why are those values set to 1?

  managedLedgerDefaultAckQuorum: '1'
  managedLedgerDefaultEnsembleSize: '1'
  managedLedgerDefaultWriteQuorum: '1'

Wouldn't these cause a data loss in case of a bookkeeper pod corruption.

BR

Matej Grasic

**Comment by lhotari** [comment]
> Hi.
> 
> We are looking at the helm production charts for Pulsar and also Bookkeeper. Why are those values set to 1?
> 
> managedLedgerDefaultAckQuorum: '1' managedLedgerDefaultEnsembleSize: '1' managedLedgerDefaultWriteQuorum: '1'
> 
> Wouldn't these cause a data loss in case of a bookkeeper pod corruption.
> 
> BR
> 
> Matej Grasic

Valid point. The defaults are not optimal in the Apache Pulsar Helm chart. Would you like to contribute a Pull Request to https://github.com/apache/pulsar-helm-chart ? Setting these values to `2` would be a more sensible default.
